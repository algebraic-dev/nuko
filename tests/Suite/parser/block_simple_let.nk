let a: Int =
    2
    3
    4