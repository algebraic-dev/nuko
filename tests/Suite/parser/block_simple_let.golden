✓ That
 └Program
   ├typeDecls: Empty List
   ├letDecls:
   | └List
   |   └LetDecl
   |     ├declName: (Name "a" 0:4-0:5)
   |     ├declArgs: Empty List
   |     ├declBody:
   |     | └Block
   |     |   ├BlBind
   |     |   | ├Lit NoExt (LInt 1:4-1:5 2)
   |     |   | └BlBind
   |     |   |   ├Lit NoExt (LInt 2:4-2:5 3)
   |     |   |   └BlEnd (Lit NoExt (LInt 3:4-3:5 4))
   |     |   └1:4-3:5
   |     ├declRet: Nothing
   |     └declExt: NoExt
   ├impDecls: Empty List
   └programExt: NoExt
